### PR TITLE
Solving the tasks for Module #4

### PR DESCRIPTION
**Pull request by NatalliaBileichyk**

### CODE REVIEW

File: module-4/longestString.js [hunk 1]
```diff
@@ -5,4 +5,32 @@
...
+                    longest = newArray[i]
+                } else if (longest.length === newArray[i].length){
+                    let sortArray = [longest, newArray[i]];
+                    longest = sortArray.sort()[0];
+                } else {
+                    longest = longest
+                }
+            }
+    } else {
+        longest = ""
```
Comment (rakur) [inline module-4/longestString.js]: At this point longest is already an empty string

File: module-4/romanToDec.js [hunk 2]
```diff
@@ -6,4 +6,79 @@
  * @param {string} roman
  * @returns {number} the Arabic (decimal) equivalent of the parameter
  * @throws Error in case of invalid input
- */
\ No newline at end of file
+ */
```
Comment (rakur) [inline module-4/romanToDec.js:9]: That's not exactly how roman numerals work, XVII should be 17 for example, IV=4, etc

File: module-4/longestString.js (hunk 1 above)
Comment (NatalliaBileichyk) [reply module-4/longestString.js]: done

File: module-4/romanToDec.js (hunk 2 above)
Comment (NatalliaBileichyk) [reply module-4/romanToDec.js:9]: A new solution has been added.

Comment (rakur) [reply module-4/romanToDec.js:9]: Could you please create a separate test file, based on the other exercises

Comment (NatalliaBileichyk) [reply module-4/romanToDec.js:9]: Done

**Review by rakur — APPROVED** [review]

**Pull request merged**